rebuild-initrd.yml

```diff
@@ -21,17 +21,25 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
+#!/usr/bin/env ansible-playbook
 ---
-dependencies: []
+- name: setup cfs_image host group
+  gather_facts: false
+  hosts: all
+  tasks:
+    - name: Add image customization hosts to the cfs_image host group.
+      add_host:
+        name: '{{ inventory_hostname }}'
+        groups: cfs_image
+      when: cray_cfs_image
```

Comment:
This use of `cray_cfs_image`, while allowed, is discouraged. See here. file:///Users/jasonsollom/working/git/docs-csm/operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md
Consider whether this can be incorporated into the hosts: line instead.